requirements.txt

```diff
@@ -1,4 +1,3 @@
-graphviz
 networkx
```

Comment:
I suppose we should indicate the version of networkx it works with (just to avoid later issues when it invariably gets updated and breaks stuff)

Reply:
Yeah, kind of makes sense, if we get more users (let's say in a few months ;) , we should think about this more. You also want to support a range of versions for dependencies I guess.